recipes/keygen-py/bld.bat

```diff
@@ -0,0 +1,5 @@
+maturin build --release
```

Comment:
Why don't you directly call `pip install` here?

Reply:
Actually we need to first build the rust bindings before installing them with `pip install`. This is why we call `maturin build` first. If we were to call `pip install .` instead, it would fail as the rust bindings are missing.

Reply:
Nearly all other `maturin` based packages succeed with a `pip install` (which itself calls maturin with some additional parameters), so this should not be the case.